xbmc/addons/AddonInstaller.cpp

```diff
@@ -714,7 +714,7 @@ bool CAddonInstallJob::Install(const std::string &installFrom, const RepositoryP
   SetText(g_localizeStrings.Get(24079));
   auto deps = m_addon->GetDependencies();
 
-  unsigned int totalSteps = static_cast<unsigned int>(deps.size());
+  unsigned int totalSteps = static_cast<unsigned int>(deps.size())+1;
```

Comment:
can you fix the formatting?
```suggestion
  unsigned int totalSteps = static_cast<unsigned int>(deps.size()) + 1;
```

Reply:
sure. each space will costs you €1 ;p